packages/account/src/Components/account-limits/__tests__/account-limits-footer.spec.tsx

```diff
@@ -36,7 +36,7 @@ describe('<AccountLimitsFooterPortal/>', () => {
     });
 
     it('should render AppSettings.Footer components correctly', () => {
-        const { container } = render(<AccountLimitsFooterPortalComponent />);
-        expect(container.getElementsByClassName('dc-app-settings__footer').length).toBe(1);
+        render(<AccountLimitsFooterPortalComponent />);
+        expect(screen.queryByTestId('dt_app_settings_footer')).toHaveClass('dc-app-settings__footer');
     });
```

Comment:
Isn’t it better to check with text rather then with classname?

Reply:
True, will edit to check it's text 🙏